app/views/orders/_order.html.erb

```diff
@@ -0,0 +1,17 @@
+<% if @customer.present? %>
```

Comment:
The logic works here since you were sending over a `@customer` value in the controller but you should be able to just go with `order.customer` directly. 

Suuuuuper side note it's possible depending on the gems you have here or in the future that you might to have an `includes` part of the `set_order` query to speed up loading, but unless you're getting an error about it I wouldn't worry about it yet. 😅